src/util/Arrays.ts

```diff
@@ -15,4 +15,19 @@ export namespace Arrays {
     export function flatten<T>(arr: T[][]): T[] {
         return arr.reduce((acc, x) => acc.concat(x), []);
     }
+
+    // Given an array, return all elements that do and do not satisfy the
+    // predicate
```

Comment:
why is this comment split over two lines? it's shorter than the code line beneath it so I don't see any reason to break it up

Reply:
I generally adhere to wrapping comment blocks at 80 columns, regardless of code wrapping. How do you want comments/code wrapped?

Reply:
I prefer to avoid wrapping in the middle of a phrase/sentence, but it's not a big deal.